ddtrace/tracer/context_test.go

```diff
@@ -80,6 +80,33 @@ func TestStartSpanFromContext(t *testing.T) {
 	assert.Equal("/", got.Resource)
 }
 
+func TestStartSpanFromContextWithSpanLinks(t *testing.T) {
+	_, _, _, stop := startTestTracer(t)
+	defer stop()
+	spanLink := ddtrace.SpanLink{TraceID: 789, TraceIDHigh: 0, SpanID: 789, Attributes: map[string]string{"reason": "terminated_context", "context_headers": "datadog"}, Flags: 0}
+	spanLinkContext := &spanContext{spanID: 789, traceID: traceIDFrom64Bits(789), spanLinks: []ddtrace.SpanLink{spanLink}}
```

Comment:
No need to store the ddtrace.SpanLink as a var if you won't reuse it. E.g,
```suggestion
	sctxWithLink := &spanContext{spanID: 789, traceID: traceIDFrom64Bits(789), spanLinks: []ddtrace.SpanLink{ddtrace.SpanLink{TraceID: 789, TraceIDHigh: 0, SpanID: 789, Attributes: map[string]string{"reason": "terminated_context", "context_headers": "datadog"}, Flags: 0}}}
```

Reply:
I use it again in the assertions to check that the correct span link is being outputted